src/frameworks/ez.md

```diff
@@ -18,4 +18,129 @@ In particular, see:
  
  * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
  * The [.platform](https://github.com/ezsystems/ezplatform/tree/master/.platform) directory
- * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically enable Redis-based cache and session support if detected.
+ * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically  enable Redis-based cache and session support if detected.
+
+## Local Development with eZ Platform 2.x and later
+
+eZ Systems provide a tool called [eZ Launchpad](https://ezsystems.github.io/launchpad/) for local development on top of a Docker stack. It improves Developer eXperience and reduces complexity for common actions by simplifying your interactions with Docker containers. eZ Launchpad is ready to work with Platform.sh.
+
+### eZ Launchpad installation
+
+eZ Launchpad's approach is to stay as decoupled as possible from your development machine and your remote hosting whether you are Linux or Mac OSX. To install run:
+
+```bash
+curl -LSs https://ezsystems.github.io/launchpad/install_curl.bash | bash
+```
+ 
+Then you can start to use it to initialize your eZ Platform project on top Docker.
+
+```bash
+~/ez init
+```
+
+or create the Docker stack based on an existing project
+
+```bash
+git clone yourproject.git application
+cd application
+~/ez create
+```
+
+You will find more details on the [eZ Launchpad documentation](https://ezsystems.github.io/launchpad/).
+
+At this time you will have a working eZ Platform application with many services including Varnish, Solr, Redis etc.
+
+### Platform.sh integration
+
+To generate the key files for Platform.sh (`.platform.app.yaml` and `.platform`) run:
+
+```bash
+~/ez platformsh:setup
+```
+
+eZ Launchpad will generate the files for you and you are then totally free to fine tune them.
+
+#### Solr specificity
+
+Solr is fully functional with eZ Launchpad but its auto-configuration for Platform.sh is not currently managed. You will have to set it up manually following the current documenation here: https://github.com/ezsystems/ezplatform/blob/master/.platform/services.yaml#L37.
+
+Actions needed are:
+
+ * generare the Solr configuration thanks to the script provided by eZ Systems
+ * put the result in the `.platform` at the root of your project
+ * add the service in the `.platform/services.yaml`
+ * add the relationship in the `.platform.app..yaml`
```

Comment:
This list should not be indented.  The bullet points are sufficient.

Also, since each line is a complete sentence it should be capitalized and end with a period.